test/python/WMCore_t/WebTools_t/REST_t.py

```diff
@@ -360,7 +360,7 @@ def testListTypeArgs(self):
         # 2 positional args (e.g. url/arg1/arg2)
         url = self.urlbase + 'listTypeArgs?aList=1'
         response = makeRequest(url=url)
-        assert response[1] == 200 and response[0] == "[1]", \
+        assert response[1] == 200 and response[0] == b"[1]", \
```

Comment:
This means that there is a change in the interface. `makeRequest` returns a list whose first position contains a bytes string in both py2 and py3 and not a "native" string (bytes in py2, uniode in py3).

Nothing to discuss really, just something that we should be aware of and maybe report somewhere. in the function docstring? in the PR description?

Reply:
updated the docstring